python/cuml/test/test_linear_model.py

```diff
@@ -111,3 +116,27 @@ def test_linear_models(datatype, X_type, y_type,
                            1e-1, with_sign=True)
         assert array_equal(skridge_predict, curidge_predict,
                            1e-1, with_sign=True)
+
+
+@pytest.mark.parametrize('num_classes', [2, 10])
+@pytest.mark.parametrize('dtype', [np.float32, np.float64])
+def test_logistic_regression(num_classes, dtype):
+    nrows = 400000
+    train_rows = np.int32(nrows*0.8)
+    X, y = make_classification(n_samples=nrows, n_features=num_classes,
+                               n_redundant=0, n_informative=2)
+
+    X_test = np.asarray(X[train_rows:, 0:]).astype(dtype)
+    X_train = np.asarray(X[0:train_rows, :]).astype(dtype)
+    y_train = np.asarray(y[0:train_rows, ]).astype(dtype)
+
+    culog = cuLog()
+    culog.fit(X_train, y_train)
+
+    sklog = skLog()
+    sklog.fit(X_train, y_train)
+
+    preds = culog.predict(X_test)
+    skpreds = sklog.predict(X_test)
+
+    assert np.sum(preds.to_array() != skpreds)/80000 < 1e-1
```

Comment:
Maybe, we should compare the cuml's coef_ with skl's here.

Reply:
that is compared already in the QN tests themselves